### ISSUE TITLE
about fine tune and train strategy

### ISSUE THREAD

**Issue by drtrista** [zhreshold/mxnet-ssd#102]
Hello! I have two questions.

1, my dataset has more class numbers  than VOC, is it possible to  use  your trained model "vgg16_ssd_300_voc0712_trainval.zip"  to fine tune my data?    How to do it? 

2, during the training,  the network needs to adjust  both the prediction convolution layers and the feature extraction layers according to the loss.  Is there any strategy during the adjusting procedure?  I mean, if the two kinds of layers are adjusting together, maybe they will influence each other so that we can not get a fast converging result? is it more reansonable that firstly fix the feature extraction layer and tune the prediction layer and then fix the prediction layer and tune the feature extraction layer? 

Looking forward to your response. Thank you very much!

**Comment by zhreshold** [comment]
Fine-tuning is simply calling train.py with --finetune, make sure you fix early layers if you training data is not large enough.

Training jointly is better IMO, but you can definitely try it. Fixing parameters is easy by set --freeze=...
